src/models/keyframesmodel.cpp

```diff
@@ -599,12 +599,72 @@ void KeyframesModel::removeAdvancedKeyframes()
             QString name = m_metadata->keyframes()->parameter(i)->property();
             if (m_filter->keyframeCount(name) > 0) {
                 m_filter->set(name, m_filter->get(name, 0));
+                for (auto& key : m_metadata->keyframes()->parameter(i)->gangedProperties()) {
+                    m_filter->set(key, m_filter->get(key, 0));
+                }
             }
         }
         reload();
     }
 }
 
+bool KeyframesModel::simpleKeyframesInUse()
+{
+    if (m_filter && m_metadata && (m_filter->animateIn() > 0 || m_filter->animateOut() > 0))
+        return true;
+    return false;
```

Comment:
This can be simplified to
`return m_filter && m_metadata && (m_filter->animateIn() > 0 || m_filter->animateOut() > 0);`

Reply:
Done